inventory/development

```diff
@@ -1 +1,2 @@
-vagrant ansible_ssh_host=172.30.1.5 ansible_ssh_user=vagrant
+vagrant ansible_ssh_host=172.30.1.5 ansible_ssh_user=vagrant ansible_ssh_private_key_file=.vagrant/machines/default/virtualbox/private_key ansible_ssh_common_args='-o StrictHostKeyChecking=no'
```

Comment:
This hasn't been necessary for me to date. When do we need this configuration?

Reply:
We don't need it when running `vagrant up` but it seems we do need if if we want to do `$ ansible-playbook playbook.yml  -i inventory/development` to update the vagrant box after its been provisioned.

Reply:
I saw in your documentation commits the suggestion to use `vagrant provision`. I wonder if I should just delete this inventory file too?

Reply:
Even better!